<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Start_Stop</class>
 <widget class="QWidget" name="Start_Stop">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1143</width>
    <height>491</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QGridLayout" name="controls_layout">
     <item row="0" column="1">
      <widget class="QPushButton" name="stop_button">
       <property name="text">
        <string>Stop</string>
       </property>
       <property name="icon">
        <iconset resource="../images/svg_resources.qrc">
         <normaloff>:/stop/stop.svg</normaloff>:/stop/stop.svg</iconset>
       </property>
      </widget>
     </item>
     <item row="0" column="3">
      <widget class="QLCDNumber" name="bias_voltage_lcd">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>30</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>30</height>
        </size>
       </property>
       <property name="smallDecimalPoint">
        <bool>false</bool>
       </property>
       <property name="segmentStyle">
        <enum>QLCDNumber::Filled</enum>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QPushButton" name="quit_button">
       <property name="text">
        <string>Quit</string>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QPushButton" name="start_button">
       <property name="text">
        <string>Start</string>
       </property>
       <property name="icon">
        <iconset resource="../images/svg_resources.qrc">
         <normaloff>:/play/start.svg</normaloff>:/play/start.svg</iconset>
       </property>
      </widget>
     </item>
     <item row="0" column="2">
      <widget class="QLabel" name="bias_voltage_label">
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>25</height>
        </size>
       </property>
       <property name="text">
        <string>Bias Voltage:</string>
       </property>
      </widget>
     </item>
     <item row="3" column="0" colspan="4">
      <widget class="QScrollArea" name="scrollArea">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>300</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="sizeAdjustPolicy">
        <enum>QAbstractScrollArea::AdjustToContents</enum>
       </property>
       <property name="widgetResizable">
        <bool>true</bool>
       </property>
       <property name="alignment">
        <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
       </property>
       <widget class="QWidget" name="scrollAreaWidgetContents">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>0</y>
          <width>1121</width>
          <height>377</height>
         </rect>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <property name="spacing">
          <number>0</number>
         </property>
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item>
          <widget class="QLabel" name="event_log">
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>0</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>16777215</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="font">
            <font>
             <pointsize>8</pointsize>
            </font>
           </property>
           <property name="text">
            <string>TextLabel</string>
           </property>
           <property name="textFormat">
            <enum>Qt::RichText</enum>
           </property>
           <property name="alignment">
            <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
           </property>
           <property name="wordWrap">
            <bool>true</bool>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </widget>
     </item>
     <item row="1" column="2">
      <widget class="QLabel" name="state_label">
       <property name="text">
        <string>Current State:</string>
       </property>
      </widget>
     </item>
     <item row="1" column="3">
      <widget class="QLabel" name="state_indi">
       <property name="font">
        <font>
         <family>Perpetua Titling MT</family>
         <pointsize>9</pointsize>
         <weight>75</weight>
         <bold>true</bold>
         <kerning>true</kerning>
        </font>
       </property>
       <property name="text">
        <string>Idle</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="event_log_label">
       <property name="text">
        <string>Event Log:</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QLabel" name="progress_label">
       <property name="text">
        <string>Progress:</string>
       </property>
      </widget>
     </item>
     <item row="2" column="2" colspan="2">
      <widget class="QProgressBar" name="progressBar">
       <property name="value">
        <number>24</number>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../images/svg_resources.qrc"/>
 </resources>
 <connections/>
</ui>
